scalapact-core/src/test/scala/com/itv/scalapactcore/common/JsonEqualitySpec.scala

```diff
@@ -151,5 +150,78 @@ class JsonEqualitySpec extends FunSpec with Matchers {
 
   }
 
+  describe("Specific cases") {
+
+    it("should cope with case #1") {
```

Comment:
This is a specific case we are seeing at work, where we thought it was something to do with the regex matching.